### PR TITLE
DoltHub.Dolt version 0.37.9

### PR DESCRIPTION
**Pull request by coffeegoddd**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56572)

### CODE REVIEW

File: manifests/d/DoltHub/Dolt/0.37.9/DoltHub.Dolt.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://github.com/dolthub/dolt/releases/download/v0.37.9/dolt-windows-amd64.msi
+  InstallerSha256: 4A4F1222F3A717B4600169B24EE9D1388A969C8C1FEFC93B9382DA5B4C1EA07E
```
Comment (Trenly) [inline manifests/d/DoltHub/Dolt/0.37.9/DoltHub.Dolt.installer.yaml:15]:
```suggestion
  InstallerSha256: 870675E7D09BEF6CFABE20AAF787D6EC4D205316AFD209F24DE3ACD89C23DCBB

```

**Review by Trenly — COMMENTED** [review]
* Duplicate of #56582

**Review by Trenly — CHANGES_REQUESTED** [review]
Hash is outdated

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/56572#issuecomment-1086875570

**Pull request closed without merging**